frontend/packages/console-plugin-sdk/src/typings/nav-items.ts

```diff
@@ -18,6 +18,10 @@ namespace ExtensionProperties {
     mergeBefore?: string;
   }
 
+  export interface SeparatorNavItem extends NavItem {
+    componentProps: NavItem['componentProps'];
```

Comment:
This isn't right. We don't need `name`.
The props here should be overridden so that you don't need to change the NavLinkProps.